tools/db/docker/.gitignore

```diff
@@ -0,0 +1 @@
+import
```

Comment:
we've tried to have just one gitignore file at the top level of the repo.
ok to add there?